### DISCUSSION TITLE
VS2022 / VCPKG 2024-06-10 / C++.  How to get F12 document browsing working for vcpkg installed header files?

### DISCUSSION THREAD

**Discussion by g40** [microsoft/vcpkg#41426] [Q&A] [answered]
I have finally got vcpkg management integrated with an existing project using the installation package [0].  

However the F12 'Goto document' does not work for .h files added via .vcpkg. 

How can this be fixed?

Many thanks.

[0]
![image](https://github.com/user-attachments/assets/dbd085ce-945f-46d1-ac46-560a61d3cbe6)

The libraries used are listed in vcpkg.json [1]. 
The relevant .h and .lib files are built etc as expected [2]

[1]
```
{
	"builtin-baseline": "2ddc10c8afa00a762a69f21f6427849fe906ede0",
	"dependencies": [
		"zeromq",
		"libssh2",
		"mbedtls",
		"libsodium"
	]
}
```

[2]
![image](https://github.com/user-attachments/assets/d6153eb4-bd00-434a-892f-550f035eb042)

[3]
![image](https://github.com/user-attachments/assets/64dcb6df-981a-46cf-b4a5-2e676cb270f4)

![image](https://github.com/user-attachments/assets/7337a13a-3792-4f33-9f79-88ddf859b8a6)

**Comment by AugP** [comment]
I don't think Go to Document generally works for external dependencies to Visual Studio. The VS project system doesn't know to look for them.

**Comment by g40** [comment, marked as answer]
Well, eventually I persuaded the F12 system to work by retyping the #include <filename>. It is a nuisance there is no GUI tooling for vcpkg in VS2022.
